src/matrix/room/timeline/entries/reply.js

```diff
@@ -72,3 +75,14 @@ export function createReplyContent(entry, msgtype, body) {
     const newFormattedBody = formattedFallback + htmlEscape(body);
     return _createReplyContent(entry.id, msgtype, newBody, newFormattedBody);
 }
+
+function createThreadContent(entry, msgtype, body) {
+    return {
+        msgtype,
+        body,
+        "m.relates_to": {
+            "rel_type": "m.thread",
+            "event_id": entry.id 
```

Comment:
Should this perhaps be `entry.content["m.relates_to"].event_id`? [MSC 3440](https://github.com/matrix-org/matrix-doc/blob/gsouquet/threading-via-relations/proposals/3440-threading-via-relations.md) specifies:
```
"m.relates_to": {
  "rel_type": "m.thread",
  "event_id": "$thread_root"
}
```